src/main/java/org/commonjava/indy/service/tracking/model/ApplicationStatus.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2023 Red Hat, Inc. (https://github.com/Commonjava/indy-tracking-service)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.tracking.model;
+
+public enum ApplicationStatus
+{
```

Comment:
Do we need to create this by ourselves for the specific thing ? I remembered that there are some 3rd libraries for this.

Reply:
Agree. I remember that jaxrs provides one: javax.ws.rs.core.Response.Status. I think we can reuse it directly.

Reply:
updated to use ` javax.ws.rs.core.Response.Status`